libs/sdk-ui-charts/src/charts/headline/internal/interfaces/ComparisonHeadlines.ts

```diff
@@ -0,0 +1,13 @@
+// (C) 2023 GoodData Corporation
+import { CalculationType } from "../../../../interfaces/index.js";
+
+export interface ICalculationDefaultValue {
+    label: string;
+    format?: string;
+}
+
+export const CALCULATION_VALUES_DEFAULT: { [key in CalculationType]?: ICalculationDefaultValue } = {
+    [CalculationType.CHANGE]: { label: "Change", format: "#,##0%" },
+    [CalculationType.RATIO]: { label: "of", format: "#,##0%" },
+    [CalculationType.DIFFERENCE]: { label: "Difference" },
```

Comment:
Please, extract these texts to localization bundle.

Reply:
I noticed that on other place you have TODO for this but please fix it now. The code will get into SDK, could get released, people could use it and it would not be translated. Ideally, the code should be delivered to SDK e2e, use case by use case.

Reply:
Updated!